pkg/features/list.go

```diff
@@ -105,7 +105,7 @@ var (
 	MicrosoftSentinelNotifier = registerFeature("Enable the Microsoft Sentinel notifier", "ROX_MICROSOFT_SENTINEL", enabled)
 
 	// ScanScheduleReportJobs enables support for compliance scan schedule report jobs
-	ScanScheduleReportJobs = registerFeature("Enables support for compliance scan schedule report jobs", "ROX_SCAN_SCHEDULE_REPORT_JOBS")
+	ScanScheduleReportJobs = registerFeature("Enables support for compliance scan schedule report jobs", "ROX_SCAN_SCHEDULE_REPORT_JOBS", enabled)
```

Comment:
Disabling this flag seems to influence only selected parts of a more complex logic. Based on the comment, it disables the "schedule" part of the reporting, but looking at the code, it does some more fine-granular things. Could we maybe explain in the comment here which precise behavior is controlled by this flag?

Reply:
The idea is that if the FF is disabled the automatic detection is disabled plus the persisting the reports. We can still send emails on demand but the status will not be persisted (just like 4.5 behavior).

Reply:
I am just worried that we may not remember in the future what was "the 4.5 behavior" and the only way of extracting what this flag does would be to follow the 9 places in the non-test code that check status of this FF.